easybuild/easyblocks/generic/modulerc.py

```diff
@@ -77,9 +72,21 @@ def make_module_step(self, fake=False):
         alias_modname = deps[0]['short_mod_name']
         self.log.info("Adding module version alias for %s to %s", alias_modname, modulerc)
 
-        module_version_specs = {'modname': alias_modname, 'sym_version': self.version, 'version': deps[0]['version']}
-        modulerc_txt = self.module_generator.modulerc(module_version=module_version_specs)
-        write_file(modulerc, modulerc_txt, backup=True)
+        # add symlink to wrapped module file when generating .modulerc in temporary directory (done during sanity check)
+        # this is strictly required for Lmod 6.x, for which .modulerc and wrapped module file must be in same location
+        if fake:
```

Comment:
Doesn't it make sense to add this symlink after the file has been created, instead of creating a broken symlink that is "fixed" afterwards?

Reply:
This is not a broken symlink at all. It links to the `Java/1.8.0_181.lua` module file (for example), which must already exist at this point (since it's a dependency for `Java/1.8`)